app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] [OPTIONAL] I had to disable the `:confirmable` module before I could un the application properly because of the errors I encountered which is a routing error. This happened while trying to do my email confirmation after signing up.
<img width="742" alt="Screenshot 2023-04-28 at 21 59 23" src="https://user-images.githubusercontent.com/58692590/235264791-b1a0ce62-ab3d-400a-8fc1-e5f4ac141c33.png">
- It will be nice if devise configuration can be checked all over again because this doesn't throw any error in the terminal.  Also, remove the `:confirmable` form from the commented part of the code on line 3 above.

Reply:
Thanks for your feedback. We'll work on it